src/pkg/src/lib.rs

```diff
@@ -389,12 +394,12 @@ impl Pkg {
                     status(status_name.as_ptr(), size, 0, ud);
 
                     // Open destination file.
-                    let mut dest = std::fs::OpenOptions::new();
-
-                    dest.create_new(true);
-                    dest.write(true);
-
-                    let mut dest = match dest.open(&output) {
+                    let mut dest = match OpenOptions::new()
+                        .write(true)
+                        .truncate(true)
+                        .create(true)
+                        .open(&output)
+                    {
```

Comment:
The reason we don't use truncate here because we want to catch if there are any duplicated file.